### PR TITLE
Fix imports and spelling errors

### PR DESCRIPTION
**Pull request by ArthurAllshire**

### CODE REVIEW

File: robot.py [hunk 1]
```diff
@@ -7,9 +7,10 @@
 
 from components.chassis import Chassis
 from components.bno055 import BNO055
-from components.gearalignmentdevice import GearAligmentDevice
+from components.gearalignmentdevice import GearAlignmentDevice
 from components.geardepositiondevice import GearDepositionDevice
 from components.winch import Winch
+from automations.manipulategear import ManipulateGear
```
Comment (auscompgeek) [inline robot.py]: unused import?

Comment (ArthurAllshire) [inline robot.py]: Oops, should be used, will amend commit

Comment (ArthurAllshire) [inline robot.py]: Fixed!

Comment (auscompgeek) [inline robot.py]: I don't see any new changes

Comment (ArthurAllshire) [inline robot.py]: Force pushed to avoid messing up commit history.

Comment (auscompgeek) [inline robot.py]: the diff is still the same as before

Comment (ArthurAllshire) [inline robot.py]: Oh right, my bad, didn't re-add robot.py

**Comment by auscompgeek** [comment]
huh, TIL github's rebase rewrites the committer

**Pull request merged**